.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out this repo
+        uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with: {go-version: '^1.13'}
```

Comment:
That's a very old version, no? Do we actually depend on this version or can we bump it?

Reply:
I wanted to align it with requirements we write here: https://github.com/sanity-io/client-go?tab=readme-ov-file#requirements

Reply:
I think we could change this to require Go 1.22.

Reply:
Okay, it makes sense since folks who can not upgrade their Go version can keep an older version of client-go pinned.

I am going to upgrade Go version in a follow up PR.